governance/security-assessment-lifecycle.md

```diff
@@ -0,0 +1,55 @@
+# Introduction
+A security assessment is a review of a project at a specific point in time. 
+The robustness of the code and operational recommendations assessed is necessarily protean as the code evolves over time and bugs are identified. 
+Issues and concerns identified in a review may be subsequently mitigated, remediated or further researched; new concerns may surface over time.
+As such, any review should be consumed by the community with attention to the date of the last assessment and after reviewing all subsequent issues and commits that may be relevant.
+
+From the perspective of the project team, it is beneficial to minimize the burden of performing ongoing reviews.  
+It should be possible to refresh the assessment incrementally unless the underlying project design or architecture changes radically.
+The scope of periodic updates should be limited in cases where there are low risk changes, or greatly expanded when security problems are identified by the community (e.g. a high impact CVE).
+
+From the perspective of the (volunteer) assessment team, performing a complete reassessment would likely be onerous - especially with if there is a queue of new project assessments and limited assessor bandwidth.
```

Comment:
nit: Maybe more generically, from the perspective of SIG Security, since we have not officially defined an "assessment team"